src/OrbitGl/TimelineTicksTest.cpp

```diff
@@ -26,4 +26,31 @@ TEST(TimelineTicks, GetMajorTicks) {
                                    1 * kNanosecondsPerMinute));
 }
 
+bool CheckTicks(uint64_t start_ns, uint64_t end_ns, std::set<uint64_t> major_ticks,
+                std::set<uint64_t> minor_ticks) {
+  TimelineTicks timeline_ticks;
+  auto all_ticks = timeline_ticks.GetAllTicks(start_ns, end_ns);
+  if (all_ticks.size() != major_ticks.size() + minor_ticks.size()) return false;
+  for (auto& [tick_type, timestamp_ns] : all_ticks) {
+    if (tick_type == TimelineTicks::TickType::kMajorTick && major_ticks.count(timestamp_ns) == 0) {
+      return false;
+    }
+    if (tick_type == TimelineTicks::TickType::kMinorTick && minor_ticks.count(timestamp_ns) == 0) {
+      return false;
+    }
+  }
+  return true;
+}
+
+TEST(TimelineTicks, GetAllTicks) {
```

Comment:
This test isn't really failing, is it? `CheckTicks` only returns a bool and doesn't test any conditions.

Reply:
Yes, for sure it was not going to be flaky in the future.

Reply:
Actually, the tests were failing (I forgot 2 minor ticks). Thanks for catching it!